### ISSUE TITLE
add feature matching algorithm for validation

### ISSUE THREAD

**Issue by anastassiavybornova** [uscuni/simplification#51]
to check out:
* https://github.com/nptscot/rnetmatch
* https://github.com/NREL/mappymatch
* https://github.com/anerv/BikeDNA

**Comment by anastassiavybornova** [comment]
see PR #52

**Comment by anastassiavybornova** [comment]
@myself: redo feature matching (once COINS is done)

**Comment by jGaboardi** [comment]
@anastassiavybornova Is this still planned?

**Comment by anastassiavybornova** [comment]
yes, i will try to implement it as part of the evaluation (method comparisons), wish me luck 😅

**Comment by jGaboardi** [comment]
> yes, i will try to implement it as part of the evaluation (method comparisons), wish me luck 😅

Luck! 🍀

**Comment by anastassiavybornova** [comment]
closing this one as absolete
